explainer/use_cases.md

```diff
@@ -69,15 +69,15 @@ This use case is the fenced frames with no special restrictions on the src and n
     *   Like all use cases, the fenced frame is isolated from the embedded context via any JS window references, script access, storage access, messaging APIs etc. The fenced frame does not know the embedding site’s origin or etld+1.
 
 To begin experimenting with fenced frames in default mode, enable the following experiments in `chrome://flags`:
-* Privacy Sandbox Ads APIs
-* Enable the \`FencedFrameConfig\` constructor.
+* Privacy Sandbox Ads APIs (chrome://flags/#privacy-sandbox-ads-apis)
```

Comment:
I don't see this in Canary. Is it gone?

Reply:
Version 118.0.5939.0 has it for me. I also installed a fresh copy of Canary on my personal laptop and verified that the flags are there as well (same version).

Reply:
Oh interesting. On M117 I have to "temporarily expire M116" to see that flag. I guess it must've expired and then they quickly renewed it. Anyways, as long as it is showing up by default in >=M118 that's good.